webservices/common/views.py

```diff
@@ -95,20 +111,39 @@ def get(self, **kwargs):
                 ),
                 status_code=422,
             )
-        if len(kwargs.get("committee_id", [])) > 1:
-            query, count = self.join_committee_queries(kwargs)
-            return utils.fetch_seek_page(query, kwargs, self.index_column, count=count)
-        query = self.build_query(**kwargs)
-        count, _ = counts.get_count(self, query)
-        return utils.fetch_seek_page(query, kwargs, self.index_column, count=count, cap=self.cap)
 
     def join_committee_queries(self, kwargs):
```

Comment:
`def join_sub_queries(self, kwargs, field, secondary_field=None`)`
could replace `join_committee_queries` and `join_year_queries`